py/htm/advanced/requirements.txt

```diff
@@ -0,0 +1,4 @@
+#See http://www.pip-installer.org/en/latest/requirements.html for details
+pillow>=7.2.0
+scipy>=1.5.1
```

Comment:
nit: can you find some lower versions? For us as a lib, it's always good to keep reasonably old requirements. (but no big deal)